settings.gradle.kts

```diff
@@ -1,3 +1,7 @@
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.9.0"
```

Comment:
What does this do? Seems unused

Reply:
It's a plugin that downloads the necessary JDKs depending on the Gradle Toolchain configuration.
It is not directly related to this change, so you can revert it, but if you plan to accept contribute, I think it is better to include it.

Reply:
https://github.com/gradle/foojay-toolchains

Reply:
reverted it